R/scale-identity.r

```diff
@@ -146,7 +146,7 @@ ScaleDiscreteIdentity <- ggproto("ScaleDiscreteIdentity", ScaleDiscrete,
 
   train = function(self, x) {
     # do nothing if no guide, otherwise train so we know what breaks to use
-    if (self$guide == "none") return()
+    if (isTRUE(self$guide == "none")) return()
```

Comment:
I prefer `identical(self$guide, "none")` in this case.

Reply:
thanks, fixed.